source/MaterialXCore/Look.cpp

```diff
@@ -4,9 +4,10 @@
 //
 
 #include <MaterialXCore/Look.h>
-
 #include <MaterialXCore/Document.h>
 
+#include <unordered_set>
```

Comment:
As above, it looks like these edits to the included headers are no longer needed.